<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherTimeTableWidget</class>
 <widget class="QWidget" name="TeacherTimeTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="timeTableWidget" native="true">
      <property name="class" stdset="0">
       <string>timeTableWidget</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>396</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="exportTimeTableOfTeacherButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>导出当前课表</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="exportTimeTableOfAllTeachersButton">
     <property name="text">
      <string>导出所有课表</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
